src/content/etherscan/components/DedaubStorageShortcut/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { type FC } from 'react'
+
+import { TokenSymbol } from '@common/components'
+
+import { DEDAUB_SUPPORT_DIRECT_LIST } from '@common/constants'
+import { getImageUrl } from '@common/utils'
+
+interface Props {
+  chain: string
+  address: string
+}
+
+const DedaubStorageShortcut: FC<Props> = ({ chain, address }) => {
+  const pathname = DEDAUB_SUPPORT_DIRECT_LIST.find(
+    item => item.chain === chain
+  )?.pathname
+
+  return (
+    <>
+      {pathname && (
+        <li
+          className="nav-item snap-align-start"
+          style={{ cursor: 'pointer' }}
```

Comment:
Perhaps we should add this CSS property in a class to preserve consistency

Reply:
Thank you for your suggestion, I will review and confirm this

Reply:
Currently, within the code base there is `common.less` at `src/common/styles/common.less` and it contains the following CSS class:

https://github.com/blocksecteam/metadock/blob/722e478d530713b1a0d19dfeb5bec4c8af1fc54e/src/common/styles/common.less#L47-L49

This class is globally available without `import`.